nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractPutElasticsearch.java

```diff
@@ -67,13 +69,28 @@ public abstract class AbstractPutElasticsearch extends AbstractProcessor impleme
         .required(true)
         .build();
 
+    static final PropertyDescriptor OUTPUT_ERROR_RESPONSES = new PropertyDescriptor.Builder()
+            .name("put-es-output-error-responses")
+            .displayName("Output Error Responses")
+            .description("If this is enabled, errors will be output to the \"error_responses\" relationship.")
+            .allowableValues("true", "false")
+            .defaultValue("false")
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .build();
+
     static final Relationship REL_SUCCESS = new Relationship.Builder()
             .name("success")
             .description("All flowfiles that succeed in being transferred into Elasticsearch go here. " +
                     "Documents received by the Elasticsearch _bulk API may still result in errors on the Elasticsearch side. " +
                     "The Elasticsearch response will need to be examined to determine whether any Document(s)/Record(s) resulted in errors.")
             .build();
 
+    static final Relationship REL_ERROR_RESPONSES = new Relationship.Builder()
+            .name("error_responses")
+            .description("Elasticsearch _bulk API responses marked as \"error\" (and optionally \"not_found\") go here.")
```

Comment:
How are the`not_found` responses optional? I didn't see a property or config option.

Reply:
This depends on the `NOT_FOUND_IS_SUCCESSFUL` PropertyDescriptors in the two implementing classes (existing properties).

I've update this relationship's description along with the existing `PropertyDescriptor`s to reflect this (also corrected the `displayName`s of those properties as they were misleading).